src/binwalk/magic/archives

```diff
@@ -48,10 +48,10 @@
 >30     string      x               name: {string}%s
 
 # ZIP footer
-0       string      PK\x05\x06      End of Zip archive
+0       string      PK\x05\x06      End of Zip archive,
+>20     leshort+22  x               footer length: %d
 >20     leshort     >0
 >>20    leshort     x               \b, comment:
->>20    leshort     x               {strlen:%d}
```

Comment:
Any reason why the strlen line was removed? This limits the size of the comment string that is printed, which would seem to be important as the comment string does not appear to be null-terminated.